src/TicketService.java

```diff
@@ -12,6 +12,21 @@ public void addTicket(Ticket ticket) {
         tickets.add(ticket);
     }
 
+    private void initializeTickets() {
+        for (int i = 1; i <= 10; i++) {
+            tickets.add(new Ticket(i, "Event " + i, 100 + i, System.currentTimeMillis(), System.currentTimeMillis(), true, 'A', 10.0 * i, 50.0 * i));
+        }
+
+        public Ticket getTicketById(int ticketId){
+            for (int i = 0; i < tickets.size(); i++) {
+                Ticket ticket = tickets.get(i);
+                if (ticket.id == ticketId) {
+                    return ticket;
+                }
+            }
+            return null;
```

Comment:
It is not best practice to return null from method like that. But if you want do that maybe it's better to change return type to Optional<Ticket>